internal/resources/azure/log_analytics_workspace.go

```diff
@@ -142,11 +150,19 @@ func (r *LogAnalyticsWorkspace) BuildResource() *schema.Resource {
 	var costComponents []*schema.CostComponent
 
 	if r.SKU == skuPerGB2018 {
-		costComponents = append(costComponents, r.logDataIngestion())
+		costComponents = append(costComponents, r.logDataIngestion("Log data ingestion", r.MonthlyLogDataIngestionGB))
+
+		if r.SentinelEnabled {
```

Comment:
If sentinel is enabled does the "Log data ingestion" cost component still apply?